src/Translator/TranslationsLoaderInterface.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Yiisoft\I18n\Translator;
+
+interface TranslationsLoaderInterface
```

Comment:
It is now https://github.com/yiisoft/i18n/blob/master/src/MessageReaderInterface.php#L6 Three separate methods were introduced to make loading delayed and allow loading only part of translations that is likely to save memory and network when having many translations in database.

Reply:
Hm... okay. I think it needs to work with declensions.

What is `$context`?

Reply:
You mean plurals? Here you go: https://github.com/yiisoft/i18n/blob/master/src/MessageReaderInterface.php#L8

$context is where you pass parameters for the message. See https://www.yiiframework.com/doc/guide/2.0/en/tutorial-i18n#message-parameters